src/main/java/com/infoshareacademy/service/GameService.java

```diff
@@ -0,0 +1,97 @@
+package com.infoshareacademy.service;
+
+import com.infoshareacademy.model.Game;
+import com.infoshareacademy.model.Questions;
+import com.infoshareacademy.model.User;
+
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+
+public class GameService {
```

Comment:
dlaczego cały serwis opierasz o static? Myślisz, że jest to konieczne ?

Reply:
To była moja pierwsza myśl, ale chyba rozumiem że lepiej będzie użyć konstruktora